.github/workflows/build-test.yml

```diff
@@ -56,7 +56,7 @@ jobs:
     - name: Get cargo-make version
       id: cargo-make-version
       run: |
-        echo "::set-output name=hash::$(cargo search cargo-make | grep '^cargo-make =' | md5sum)"
+        echo "hash=$(cargo search cargo-make | grep '^cargo-make =' | md5sum)" >> $GITHUB_OUTPUT
```

Comment:
question: what does this change do?

Reply:
[`::set-output` is deprecated and will be removed](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)